<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZoomWidget</class>
 <widget class="QWidget" name="ZoomWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>42</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSpinBox" name="zoomSpinBox">
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>400</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="zoomOutButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="view_prototype.qrc">
       <normaloff>:/icon/images/zoom_out.png</normaloff>:/icon/images/zoom_out.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="zoomHorizontalSlider">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="sliderPosition">
      <number>100</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>99</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="zoomInButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="view_prototype.qrc">
       <normaloff>:/icon/images/zoom_in.png</normaloff>:/icon/images/zoom_in.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="view_prototype.qrc"/>
 </resources>
 <connections/>
</ui>
